README.md

```diff
@@ -6,7 +6,7 @@ Reference sparse coding implementations for efficient learning and inference imp
 
 ## Implemented Inference Methods
 * Locally Competative Algorithm (LCA)
-* Gradient Descent with Euler's method on Laplace Prior (Vanilla)
+* Gradient Descent with Euler's method on Laplace Prior (Euler)
```

Comment:
Curious about the rename. While Vanilla is a bit obscure, it encompasses the method (euler), objective (laplace), and historical context (original 1996 paper). Euler is almost more vague. Many inference methods could use Euler's method. Thoughts on keeping this inference method called vanilla?

Reply:
Maybe `Nature96` or `Olshausen96`